### ISSUE TITLE
Theta Guardian Node not Starting

### ISSUE THREAD

**Issue by learnedall** [thetatoken/guardian-mainnet-guide#66]
I've attempted the recommended delete database, restart node, set performance to standard, even uninstalled and reinstalled.  The node however when loading gets to "perform health check" and "start node" but it ends up restarting the loading snapshot.  What can I do to get this back up and running?

I just noted that I have had the Edge node on simultaneously.  Could that be causing problems with the load?  Appreciate the guidance.

**Comment by Mendez849** [comment]
@learnedall Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by Mendez849** [comment]
@learnedall Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by Mendez849** [comment]
@learnedall Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by Mendez849** [comment]
@learnedall Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by Mendez849** [comment]
@learnedall Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by ArmoredRage** [comment]
I have the same issue. Tryed the steps on every forum and nothing. Even uninstalled, deleted registry entries, removed all leftover files from examining profile( database, logs,etc) then restarted a clean install and still have the same loop issue

**Comment by Mendez849** [comment]
Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by Mendez849** [comment]
Hello Mate, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by learnedall** [comment]
![image](https://user-images.githubusercontent.com/2061948/110252319-a9966000-7f39-11eb-96e6-9df39dd51e97.png)

I tried what JanTheta said in the Theta Network discord but it didn't work (instructions in image below).  Looks like Theta is doing maintenance and others are experiencing the same.  Let's hope it gets resolved soon.

**Comment by Mendez849** [comment]
@learnedall @ArmoredRage Try to Message Theta Support on telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by learnedall** [comment]
Thanks Mendez but Theta Network admins don't help like that.  Not suggesting you aren't helpful, but for anyone else that sees this post, please follow the golden rules.  

:MitchLiu: Never give your private keys / mnemonic phrases to anyone for any reason ever!

:MitchLiu: If you get a DM / PM message, or a phone call, then it's scammer NO matter what they say, or how official their account looks!

**Comment by ArmoredRage** [comment]
Thanks for the post, ive tried that still no luck. Yeah I backed up my key and yaml config. I had a feeling it had somthinf to do with this 3 day long unscheduled maint that is going on. Will just have to wait it out I guess.

**Comment by TrySomeThinking** [comment]
Beginning today, my Guardian Node connects to no peers.  I can't find any block by security software and I have the node flagged as trusted.  My internet connection of great.  No peers whether VPN or not, and changing VPN address doesn't help either.  The following entries keep repeating in the log file.

[2021-03-07 18:06:48.326] [info] [2021-03-07 18:06:48]  INFO [p2p] Handshaking with 34.83.204.5:21000...

[2021-03-07 18:06:48.407] [error] [2021-03-07 18:06:48] ERROR [p2p] Error during handshake/recv: rlp: expected input list for types.NodeInfo
[2021-03-07 18:06:48.408] [error] [2021-03-07 18:06:48] ERROR [p2p] Failed to handshake with peer, error: rlp: expected input list for types.NodeInfo
[2021-03-07 18:06:48.408] [info] [2021-03-07 18:06:48] ERROR [p2p] Error during handshake/recv: rlp: expected input list for types.NodeInfo
[2021-03-07 18:06:48] ERROR [p2p] Failed to handshake with peer, error: rlp: expected input list for types.NodeInfo

**Comment by SchaffJ** [comment]
I’m in the same boat— basically the same logs. Yesterday I had 96 peers, this morning at 4am I noticed 24, 8am 3 and since 8:30am none. Last reward was 8:27 am  My local network is at 9,461,800 blocks.

**Comment by TrySomeThinking** [comment]
Additional info: My local network is at 9,463,614 and holding, unable to connect to any peer.  Running version 2.2.4.  Restarting the node, resetting peers and rebooting the computer do nothing to help the problem.

**Comment by Mendez849** [comment]
@SchaffJ @TrySomeThinking Hello Mates, Sorry to hear about your issue, you can contact Theta Network Support on Telegram via the weblink www.t.me/thethetanetwork. They'll be happy to resolve your issue. Cheers.

**Comment by SchaffJ** [comment]
Ok thanks, I’ll give it a go.

**Comment by ArmoredRage** [comment]
Dont follow the link its a scam. Never give passwords or keys to anyone.

**Comment by SchaffJ** [comment]
Ya I realized it was was a scammer right away.... thanks for the heads up

**Comment by SchaffJ** [comment]
I did go to it, and they tried to get me to download a Trojan or some other type of malware

**Comment by SchaffJ** [comment]
Well, seems to be working now— whatever was interrupting the guardian node has been resolved with no action on my part.

**Comment by TrySomeThinking** [comment]
Yes, my node has also reconnected and resumed synchronizing.  Cause and remedy unknown.

**Comment by flashyanswer** [comment]
> Dont follow the link its a scam. Never give passwords or keys to anyone.

Hi,
Has the link been removed already or are you talking about the png showing the Jantheta advice?
Thanks,

**Comment by SchaffJ** [comment]
Yeah, it was a link that has long been removed. Thanks!

**Comment by malcypresshill** [comment]
good morning friends, my guardian never opened, he always stays on starting and doesn’t advance, he also doesn’t appear reading the screenshots as he did before, I tried to reinstall and nothing.  I also tried to reset everything in the options of the program itself and it didn't work, what am I doing wrong?  how to fix this?

![Uploading 113308400...]![image](https://user-images.githubusercontent.com/81757473/113308429-88c4ee80-92dc-11eb-8614-81b157359507.jpeg)

![image](https://user-images.githubusercontent.com/81757473/113308530-a4c89000-92dc-11eb-8433-c38e3b92cb40.jpeg)

**Comment by SchaffJ** [comment]
It can’t validate the snapshot (the starter download of the blockchain)

there’s a reset DB you can try. It’s in the menus on the guardian node.  Also, my peers are normally 96 and there are less than 20.  Last time that happened the issue resolved itself.

**Comment by bugsbacon** [comment]
My guardian node went dow yesterday 31st of may. Now it won’t restart. Is there a fix for this? Theta support have yet to reply

**Comment by Adam-Jack12** [comment]
@bugsbacon Has your issue been resolved ?

**Comment by bugsbacon** [comment]
No not yet. 

Still getting spinning wheel when it says starting node

**Comment by Adam-Jack12** [comment]
Sorry to hear about your issue, you can contact the Theta Network Support on Telegram via the weblink https://t.me/thetanetworking

 They'll be happy to resolve your issue. Cheers.

**Comment by Adam-Jack12** [comment]
@bugsbacon Sorry to hear about your issue, you can contact the Theta Network Support on Telegram via the weblink https://t.me/thetanetworking

They'll be happy to resolve your issue. Cheers.

**Comment by Waldy13** [comment]
I find the same problem “infinite starting loop” today. I think THETA is doing maintenance now. No feedback from Theta support.

**Comment by Adam-Jack12** [comment]
@Waldy13 You can contact the Theta Network Support on Telegram via the weblink www.t.me/thetanetworking. They'll be happy to resolve your issue

**Comment by bugsbacon** [comment]
Yes still same issue & no word from support. There is a telegram admin group but they wanted me to click a link which I said I would not do. This is all very unsatisfactory. Theta website should have specific tech support. This is not going to encourage others to become guardian nodes

**Comment by Adam-Jack12** [comment]
@bugsbacon thats the same procedure I went through. 
They helped me with mine yesterday

**Comment by bugsbacon** [comment]
But I am not comfortable clicking their link to my web wallet. Where on theta website says this is their official tech support?

**Comment by Adam-Jack12** [comment]
@bugsbacon its fine if you don’t want to. I had my doubts too. Cheers

**Comment by SchaffJ** [comment]
Bugs— don’t do that. Adam jack is a thief and con artist.

**Comment by SchaffJ** [comment]
No one will help ever you in this manner— if you guardian node has become corrupt, it itself contains the tools to get you back up and running. You can go to node—> delete database. Do not use the option to delete your key. 

After you delete your database restart and it will get a new blockchain snapshot and start adding blocks to it until it catches up, after that it will run as normal.

**Comment by ArmoredRage** [comment]
Yes there is some testing going on due to the massive increase in nodes plua the 1mil new theta wallets in the last 30 days. Try deleting your DB and re syncing it worked for me. It will come back up. If you need assistance dont use telegram its no longer supported and never click there links. Official support is now on discord. There is usually a few admins online there that can help.

**Comment by bugsbacon** [comment]
Have tried delete database but still won’t restart

**Comment by SchaffJ** [comment]
Go here and post, do not answer and DM, even though some are users legitimately just trying to help you— there’s Adam jack types that they are weeding out as they see them.

https://discord.gg/8e7RK99B

**Comment by SchaffJ** [comment]
Sorry do not answer *any* dm that should say. Anyhow only publicly chat in the discord.

**Comment by bugsbacon** [comment]
Thanks

**Comment by Waldy13** [comment]
Dear Schaff,

I have deleted the DB a few times, restarted GN program and even restarted
my Mac. But the problem persists.
I've heard that even reinstalling the GN program, it won't work.

Thanks.

Wallace

**Comment by Waldy13** [comment]
I find a phenomenon that before the issue of "not starting", my Mac
suddenly disconnected from internet very often (for about two weeks). It is
quite strange that after the occurrence of "not starting", no internet
disconnection appear again. I have updated my Mac OS to latest version. Are
there any clues?

**Comment by SchaffJ** [comment]
I’m sorry I don’t know a ton about the Mac— can you join the discord from that link I posted earlier? And ask there?
